packages/engine/src/ecs/functions/EntityTree.ts

```diff
@@ -220,10 +214,9 @@ export function addEntityNodeChild(entity: Entity, parentEntity: Entity, childIn
   const childTransform = getComponent(entity, TransformComponent)
   getMutableState(EngineState).transformsNeedSorting.set(true)
   if (parentTransform && childTransform) {
-    computeTransformMatrix(parentEntity)
-    computeTransformMatrix(entity)
+    // computeTransformMatrix(parentEntity)
```

Comment:
is this intentionally left commented? if this is unneeded, we should hyst remove it

Reply:
removed